examples/getting_started/README.md

```diff
@@ -1405,6 +1451,30 @@ Terraform will perform the following actions:
       - organization_id = "7a99d00c-f55b-4b39-bc72-1b4cc68ba894" -> null
     }
 
+  # couchbase-capella_sample_bucket.new_sample_bucket will be destroyed
+  - resource "couchbase-capella_sample_bucket" "new_sample_bucket" {
+      - bucket_conflict_resolution = "seqno" -> null
+      - cluster_id                 = "bb96d3bb-ea30-4f9b-aafa-fef8e4db9a48" -> null
+      - durability_level           = "none" -> null
+      - eviction_policy            = "fullEviction" -> null
+      - flush                      = false -> null
+      - id                         = "Z2FtZXNpbS1zYW1wbGU=" -> null
+      - memory_allocation_in_mb    = 200 -> null
+      - name                       = "gamesim-sample" -> null
+      - organization_id            = "6af08c0a-8cab-4c1c-b257-b521575c16d0" -> null
+      - project_id                 = "7ee0ced2-a2f6-44c2-8541-07d97d9e7670" -> null
+      - replicas                   = 1 -> null
+      - stats                      = {
+          - disk_used_in_mib   = 20 -> null
+          - item_count         = 586 -> null
+          - memory_used_in_mib = 62 -> null
+          - ops_per_second     = 0 -> null
+        } -> null
+      - storage_backend            = "couchstore" -> null
+      - time_to_live_in_seconds    = 0 -> null
+      - type                       = "couchbase" -> null
+    }
+
 Plan: 0 to add, 0 to change, 7 to destroy.
```

Comment:
Don't we need to change this line everywhere in this file, as and when we add the resources?🤔